pkgs/development/python-modules/cloud-tpu-client/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pythonOlder
+, absl-py
+, google-api-python-client
+, oauth2client
+}:
+
+buildPythonPackage rec {
+  pname = "cloud-tpu-client";
```

Comment:
What's the difference between cloud-tpu-client and google-cloud-tpu?

Reply:
Sorry is `google-cloud-tpu` a package in Nix somewhere?